### DISCUSSION TITLE
高校生活质量问卷160份（持续更新）

### DISCUSSION THREAD

**Discussion by HUAWEIGI** [CollegesChat/university-information#175] [Enhancement]
项目作者您好，这是我收集的160份高校生活质量数据，希望能并入到您的项目中，为项目贡献一份力量，后续我会持续更新数据。
【腾讯文档】高校生活水平调查_数据详情表
https://docs.qq.com/sheet/DRHdLb0JhWWlnd1h3?tab=000001

**Comment by kaixinol** [comment]
你好還有更新嗎（？
我打算合并了
